ext/EnzymeExt.jl

```diff
@@ -0,0 +1,40 @@
+module EnzymeExt
```

Comment:
It's recommended to make extension names unambiguous by prefixing with the package name in the filename, module, and extension name. See https://github.com/JuliaMath/ChangesOfVariables.jl/pull/13#issuecomment-1427763012
```suggestion
module CUDAEnzymeCoreExt
```